frames/anime.lua

```diff
@@ -234,7 +234,7 @@ function _detalhes.PlayBestDamageOnGuild (damage)
 	Portrait.scale:SetStartDelay (0)
 	Portrait.scale:SetEndDelay (0)
 
-	if (DetailsFramework.IsDragonflight() or DetailsFramework.IsWotLKWowWithRetailAPI()) then
+	if (DetailsFramework.IsDragonflight() or DetailsFramework.IsNonRetailWowWithRetailAPI()) then
```

Comment:
Change the function name to the 'correct' one. Just makes for finding these functions easier.